lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -19,6 +19,7 @@
 
 #include <mayaUsd/nodes/proxyShapeBase.h>
 #include <mayaUsd/ufe/ProxyShapeHandler.h>
+#include <mayaUsd/ufe/UfeVersionCompat.h>
```

Comment:
@seando-adsk I had a closer look at the logic in UfeVersionCompat.h and agree that it doesn't have to be wrapped by ifdef.